lib/backend/dynamo/dynamodbbk.go

```diff
@@ -822,6 +828,52 @@ func (b *Backend) getKey(ctx context.Context, key []byte) (*record, error) {
 	return &r, nil
 }
 
+// getContinuousBackups gets the state of continuous backups.
+func (b *Backend) getContinuousBackups(ctx context.Context) (bool, error) {
+	resp, err := b.svc.DescribeContinuousBackupsWithContext(ctx, &dynamodb.DescribeContinuousBackupsInput{
+		TableName: aws.String(b.Tablename),
+	})
+	if err != nil {
+		return false, convertError(err)
+	}
+
+	switch *resp.ContinuousBackupsDescription.PointInTimeRecoveryDescription.PointInTimeRecoveryStatus {
+	case string(dynamodb.ContinuousBackupsStatusEnabled):
+		return true, nil
+	case string(dynamodb.ContinuousBackupsStatusDisabled):
+		return false, nil
+	default:
+		return false, trace.BadParameter("dynamo return unknown state for continuous backups: %v",
+			*resp.ContinuousBackupsDescription.PointInTimeRecoveryDescription.PointInTimeRecoveryStatus)
+	}
+}
+
+// setContinuousBackups sets the state of continuous backups.
+func (b *Backend) setContinuousBackups(ctx context.Context) error {
+	// Make request to AWS to update continuous backups settings.
+	_, err := b.svc.UpdateContinuousBackupsWithContext(ctx, &dynamodb.UpdateContinuousBackupsInput{
+		PointInTimeRecoverySpecification: &dynamodb.PointInTimeRecoverySpecification{
+			PointInTimeRecoveryEnabled: aws.Bool(b.Config.EnableContinuousBackups),
+		},
+		TableName: aws.String(b.Tablename),
+	})
+	if err != nil {
+		return convertError(err)
+	}
+
+	// Make sure the state Dynamo is now in matches the expected state.
+	status, err := b.getContinuousBackups(ctx)
```

Comment:
Why is this necessary? I'm not familiar with dynamodb but from what I read about `UpdateContinuousBackups` API it seems to ascertain that after a successful invocation response contains the `ContinuousBackDescription` which reflects the update:
```
...A successful UpdateContinuousBackups call returns the current ContinuousBackupsDescription.
Continuous backups are ENABLED on all tables at table creation. If point in time recovery is enabled, PointInTimeRecoveryStatus will be set to ENABLED...
```
Is it b/c it was found to return the result inconsistently or for other reason?

Reply:
Yeah, `ContinuousBackupsStatus` in the response always appears to have a invalid response. I could assume no error meant the call was successful, but since I had to add `getContinuousBackups` for tests, figured might as well verify the property was set correctly.

Reply:
@a-palchikov Actually I went with your suggestion, if an error is not returned I assume the operation was successful. I test for correctness in a test.